### DISCUSSION TITLE
Error : Failed to load PDF document

### DISCUSSION THREAD

**Discussion by MrAlexanderWs** [dompdf/dompdf#2381] [General]
I have just get a certificate project from this code https://github.com/YunusEmreNalbant/dynamic_certificate. When I test this project, this should be no problem for me. It display properly. And I have copy their DOMPDF to my current project

This is the output of generated certificate code.
![contoh pt 2](https://user-images.githubusercontent.com/46021397/108676340-a8693a00-751a-11eb-9ca4-2a7c135d0900.png)

This is the controller to generate a certificate from controller `Person.php`. This is from person table and name_surname column
![contoh 11](https://user-images.githubusercontent.com/46021397/108676417-c2a31800-751a-11eb-850a-5fbc10357a9b.png)

These projects doesn't have any problem

But in my current project it display error failed to load pdf document. Even I have install dompdf and inspect console and network it still empty

This is the controller to generate a certificate from controller `adm.php`, and this code is inside `public function overall()`
![contoh pt 6](https://user-images.githubusercontent.com/46021397/108676482-dea6b980-751a-11eb-9657-2b1d3f750ec8.png)

and this is the view to generate a certificate (v_sertifikat.php). This is from m_siswa and nama column
![contoh pt 7](https://user-images.githubusercontent.com/46021397/108676624-1ada1a00-751b-11eb-9f15-4dc5d1488ccc.png)

That project it won't work. It shows error failed to load PDF document

So what's the code to change to fix this pdf error? I checking Inspect console and network, it shows nothing.

This is m_siswa table
![contoh 12](https://user-images.githubusercontent.com/46021397/108676980-9cca4300-751b-11eb-94e2-6f414e6c1681.png)

This is person table from other project
![contoh 13](https://user-images.githubusercontent.com/46021397/108677083-c2574c80-751b-11eb-9135-e372935f0033.png)

**Comment by bsweeney** [comment]
Are you saying this particular project works fine, but when you try to reproduce the logic in a separate project you're unable to generate a PDF?

FYI since this doesn't necessarily look to be an issue with Dompdf itself I may not have the bandwidth to help (though I'm usually happy to try). You might want to post your question on StackOverflow or another general support forum so more people have an opportunity to help.
